.github/workflows/merge_to_main_production.yaml

```diff
@@ -5,7 +5,6 @@ on:
     branches:
       - main
     paths:
-      - ".github/workflows/merge_to_main_production.yaml"
```

Comment:
Hmm working on top of my memory, we had the lambda image ID defined in there before and  hence why this was specified? Is this right?